src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const key in action.extraData) {
+          state[key] = action.extraData[key];
+        }
+        break;
+      case 'removeProperties':
+        for (const remove of action.keysToRemove) {
```

Comment:
you're iterating through array of keysToRemove, so the value, naturally would be 
```suggestion
        for (const keyToRemove of action.keysToRemove) {
```